scripts/plotting/fakerate.py

```diff
@@ -1024,7 +1027,11 @@ def plot_closure(h, outdir, suffix="", outfile=f"closureABCD", ratio=True, proc=
         outfile += f"_{args.postfix}"
 
     plot_tools.save_pdf_and_png(outdir, outfile)
-
+    plot_tools.write_index_and_log(outdir, outfile, 
+        # yield_tables={"Stacked processes" : stack_yields, "Unstacked processes" : unstacked_yields},
+        # analysis_meta_info={"AnalysisOutput" : groups.getMetaInfo()},
```

Comment:
Don't you want to save the meta info, at least?

Reply:
This is not so important since this is only a plotting script for validation, and I save the command used to run the script.  But I can add the metadata in a future commit.